src/components/RenderGroceryById/RenderGroceryById.js

```diff
@@ -14,7 +14,7 @@ class RenderGroceryById extends Component {
         <ul>
           {gf
             .getGroceryByNameWithDepAndIng(
-              gf.getGroceryById(parseInt(this.props.match.params.id, 10))[0]
+              gf.getGroceryById((parseInt(this.props.match.params.id) - 10), 10)[0]
```

Comment:
this is a very confusing construction for me. i understand what you want to do, but it's over complicated.
instead of this skycrapper - let's split it into 4-5 separated "things". make it easy to understand for other team members - because i assume if we'll show it to some one - it'll have hard time to understand this construction

Reply:
do you mean replace 10 on 4?

Reply:
small example: 
```
let id = '';
let grocery = gf.getGroceryById(id)
let data = gf
            .getGroceryByNameWithDepAndIng( grocery );
```